src/app/StoreProvider.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import { useRef } from 'react'
+import { Provider } from 'react-redux'
+import { makeStore, AppStore } from '../redux/store'
+import { setLanguage } from '../redux/languageSlice'
+

```

Comment:
```suggestion
'use client';

import { useRef } from 'react';
import { Provider } from 'react-redux';
import { makeStore, AppStore } from '../redux/store';
import { setLanguage } from '../redux/languageSlice';

```
Возможно, это кажется незначительной деталью. Но следует соблюдать правила написания кода всегда и обращать на это внимание.
Порядок в написании кода упрощает работу вам и вашим коллегам.